pkg/jtracer/jtracer.go

```diff
@@ -45,23 +42,19 @@ func New(serviceName string) (*JTracer, error) {
 	if err != nil {
 		return nil, err
 	}
-	// Use the bridgeTracer as your OpenTracing tracer(otTrace).
-	otelTracer := tracerProvider.Tracer("github.com/jaegertracing/jaeger/pkg/jtracer")
-	otTracer, wrappedTracerProvider := otbridge.NewTracerPair(otelTracer)
 
 	closer := func(ctx context.Context) error {
 		return tracerProvider.Shutdown(ctx)
 	}
 
 	return &JTracer{
-		OT:     otTracer,
-		OTEL:   wrappedTracerProvider,
+		OTEL:   tracerProvider,
 		closer: closer,
 	}, nil
 }
 
 func NoOp() *JTracer {
```

Comment:
Is this still used?

The closer should be safe to use, a noop function, not nil.